.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 zinc/src/test/resources/bin
+.idea
```

Comment:
If you are an IntelliJ IDEA user I would suggest you configure your global gitignore file, instead of expecting every project to include these ignores or adding them while submitting a pull request.

GitHub has a nice little help article about it: https://help.github.com/articles/ignoring-files/.

The general principle is that .gitignore should only include paths for files that are generated by running the build, and not include a random assortment of paths generated other tools.